apps/emqx_management/src/emqx_mgmt_api_configs.erl

```diff
@@ -37,6 +37,7 @@
 -define(PREFIX_RESET, "/configs_reset/").
 -define(ERR_MSG(MSG), list_to_binary(io_lib:format("~p", [MSG]))).
 -define(OPTS, #{rawconf_with_defaults => true, override_to => cluster}).
+-define(TAGS, ["Configs"]).
```

Comment:
Can this be a string? The previous one was an atom.

Reply:
Both of string and atom is ok :)